common.js

```diff
@@ -1889,6 +1889,36 @@ const allSoloCars = {
       '2006': ['fs'],
     },
   },
+  'Lancia': {
+    'Beta': {
+      '1975': ['hs', 'ep', 'fsp'],
+      '1976': ['hs', 'ep', 'fsp'],
+      '1977': ['hs', 'ep', 'fsp'],
+      '1978': ['hs', 'ep', 'fsp'],
+      '1979': ['hs', 'ep', 'fsp'],
+      '1980': ['hs', 'ep', 'fsp'],
+      '1981': ['hs', 'ep', 'fsp'],
+      '1982': ['hs', 'ep', 'fsp'],
+      '1983': ['hs', 'fsp']
+    },
+    'Scorpion': {
+      'all': ['hs', 'bsp']
+    },
+    'Scorpion (1756 cc)': {
+      '1976': ['dp']
```

Comment:
This should be merged with the above `Scorpion`. There was only one kind of Scorpion sold in North America, and it had the 1756 cc engine. Just add `dp` to the existing `Scorpion` and you should be good!